### DISCUSSION TITLE
sites = 0 OnlyFans

### DISCUSSION THREAD

**Discussion by KraillerTru** [UltimaHoarder/UltimaScraper#1219] [Q&A] [answered]
I'm learning about everything, and It take me a long time to arrive to this, so I think I'm doing something wrong.
In this Step

auth.json has been created. Fill in the relevant details and then press enter to continue.

In Auth.json I Fill only:
Username
auth_id
sess
user_agent
x_bc

And with this I do nothing
"auth_hash": "",
"auth_uniq_": "",
"email": "",
"password": "",

and when a press enter appears:
**Site: 0 = onlyfans |**

So a I don't Know what to do, sorry I'm trying to learn

Some doubts
"username" I use my profile name in OnlyFans right?
And "support_2fa" can i put it in false? Because I don't use 2fa

**Comment by DannyTwice** [comment, marked as answer]
Press "0" then enter after that

**Reply by KraillerTru** [reply]
Thank you! It solved my problem

**Comment by cubcbb** [comment]
Choose Profile
Names: Username = username  | 0 = All | 1 = default
This comes up, however regardless of entry (0/1), seems unable to scrape successfully. I have input the same entry field as per your option, any idea? Thanks in advance

Auth (V1) Attempt 1/10
Access denied.
Auth (V1) Attempt 2/10
Access denied.
Auth (V1) Attempt 3/10
Access denied.
Auth (V1) Attempt 4/10
Access denied.
Auth (V1) Attempt 5/10
Access denied.
Auth (V1) Attempt 6/10
Access denied.
Auth (V1) Attempt 7/10
Access denied.
Auth (V1) Attempt 8/10
Access denied.
Auth (V1) Attempt 9/10
Access denied.
Auth (V1) Attempt 10/10
Access denied.

**Reply by LloydeSorrow** [reply]
Just had this problem myself. I had to update the session field in the auth.json file (I also added in my email and password).
